docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -118,6 +118,41 @@ There are multiple [data encryption options](https://cloud.google.com/storage/do
 - [Customer-supplied encryption keys](https://cloud.google.com/storage/docs/encryption/customer-supplied-keys) are not supported.
 - [Client-side encryption keys](https://cloud.google.com/storage/docs/encryption/client-side-keys) are not supported.
 
+### Azure backup store
+
+To store your backups in Azure Storage, choose the `Azure` backup store and specify how to connect with the Azure container:
```

Comment:
Where exactly do I choose the **Azure** backup store? This is in Azure Storage? Do I just add it into the YAML snippet as shown below? I only ask because I can't have a look at a screenshot/example 👍

Reply:
Yes, we have to set this yaml var `zeebe.broker.data.backup.store` to `AZURE`, like in the examples above.

<img width="480" alt="image" src="https://github.com/camunda/camunda-docs/assets/132340590/a7a59352-525d-4fb3-a384-3cdaba647c2e">